_posts/2021-02-00-the-life-science-community.md

```diff
@@ -7,71 +7,136 @@ theme: twitter
+* [Opportunities we see](#Opportunities-we-see)
+* [Strategic plan](#strategic-plan)
+* [Limitations](#Limitations)
+* [Methods](#Methods)
 
-Introduction
-------------
+## Background
 
-Recently Dask [won some funding](https://chanzuckerberg.com/eoss/proposals/) to hire a developer ([Genevieve Buckley]()) to improve Dask specifically for
+Recently Dask [won some funding](https://chanzuckerberg.com/eoss/proposals/) to hire a developer ([Genevieve Buckley](https://github.com/GenevieveBuckley/)) to improve Dask specifically for life sciences.
 
 Working with scientists is a really great way to drive growth in open source projects. Both scientists and software developers benefit. Early on, Dask had a lot of success integrating with the geosciences community. It'd be great to see similar success for life sciences too.
 
-
 There are several areas of life science where we see Dask being used today:
 * Biological image processing
 * Single cell analysis
 * Statistical genetics
+* ...and many more
 
-What we learned
-----------------
+We've solicited feedback from the life science community, to come up with a strategic plan to direct our effort over the next year.
 
-### Dask users
-When we talked to people, a lot of similar themes to their comments.
+## What we learned
+### From Dask users
+When we talked to individual Dask users, we heard a lot of similar themes in their comments.
 
 People wanted:
 1. Better documentation and examples
 2. Better support for working with constrained resources
 3. Better interoperability with other software tools
 
-The most common request was for better documentation with more examples. People across many different areas of life science all said this could help them a lot.
+The most common request was for better documentation with more examples. People across many different areas of life science all said this could help them a lot. A corresponding challenge here is the multitude of different areas of life science, all of which require targeted documentation.
 
 GPU support was also commonly mentioned. Comments about GPUs fit into two of the categories above: GPU memory is often a constraint, and life scientists also want it to be easier to apply deep learning models to their data.
 
-### Dask with other software
+### From other software libraries
 We didn't only talk with individual users of Dask. We also spoke to developers of scientific software projects.
 
 Some of the software projects we spoke to include:
+* [sgkit](https://github.com/pystatgen/sgkit)
 * [scanpy](https://scanpy.readthedocs.io/en/stable/)
+* [squidpy](https://github.com/theislab/squidpy)
 * [ilastik](https://www.ilastik.org/)
 * [CellProfiler](https://cellprofiler.org/)
 * [napari](https://napari.org/)
 
 Some projects, like [napari](https://napari.org/), already use Dask under the hood. Other projects don't use Dask at all, but think that it might be able to help them.
 
 Software projects wanted to solve problems related to:
```

Comment:
I think moving this list on how dask can help other libaries/project to a location earlier in this section would be helpful.
1) Why adopt Dask ? Because 
2) Who we've talked to
3) Current status